src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -628,6 +628,9 @@ private void checkClosed() throws SQLServerException {
                 }
             }
         }
+        catch (SQLServerException e) {
+            throw e;
```

Comment:
Just add one line comment like `SQLServerException should not again wrapped by SQLServerException`